pkg/nimble/README.ipv6-over-ble.md

```diff
@@ -0,0 +1,157 @@
+    sudo su
+
+    # Mount debugfs file system, already the case on many distributions
+    mount -t debugfs none /sys/kernel/debug
+
+    # Load 6LoWPAN module
+    modprobe bluetooth_6lowpan
+
+    # Enable the bluetooth 6lowpan module.
+    echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
+
+Next, we should verify that our Bluetooth device is configured properly:
+
+    # Look for available HCI devices.
+    hciconfig
+
+This should show us some information about the available Blutooth devices. If no
+device is listed here, something is wrong...
+
+
+### Connecting to the RIOT node
+
+When connecting to the RIOT device, you need to know the BLE address of the
+device. There are two ways to find out:
+
+a) use `ble info` or `ifconfig` on the RIOT device directly and copy the address
+
+b) use `bluetoothctl` on Linux to scan for the device. Once `bluetoothctl` has
+   started, issue `scan on` to start scanning. The default name for the RIOT
+   device is set to `RIOT-GNRC`, so you should see it pop up.
+
+c) use `hcitool lescan` on the Linux host (older tool, predecessor of
+   `bluetoothctl`...)
+
+Once you have the address, you simply connect Linux to RIOT using the following
+command:
+
+    # Put your device address here...
+    # Note: the 2 after the address denotes a BLE public random address, default
+    #       used by `nimble_netif`
+    echo "connect UU:VV:WW:XX:YY:ZZ 2" > /sys/kernel/debug/bluetooth/6lowpan_control
+
+Thats it, you now have a IPv6 connection to your RIOT node. You can verify this
+using `ifconfig`, where something like the following should be visible:
+
+    bt0       Link encap:UNSPEC  HWaddr 00-19-86-00-16-CA-00-00-00-00-00-00-00-00-00-00
+          inet6 addr: fe80::19:86ff:fe00:16ca/64 Scope:Link
+          UP RUNNING MULTICAST  MTU:1280  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:18 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:1000
+          RX bytes:0 (0.0 B)  TX bytes:1624 (1.6 KB)
+
+You should also be able to ping your RIOT node. The devices link local address
+is also printed when running `ble info` on your RIOT device.
+
+    # Substitute the actual device address,
+    ping6 fe80::uuvv:wwff:fexx:yyzz%bt0
+
+Now everything should be fine :-)
+
+
+## [optional] Distributing a routeable Prefix
+
+You can use the Router Advertisement Deamon (`radvd`) in Linux to automatically
+distribute prefixes in your BLE network. For the following, you need to make
+sure that `radvd` is installed on your Linux host.
+
+As a first step, we need to enable IPv6 forwarding in Linux:
+
+    sudo echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
+
+Next, we configure `radvd` (`etc/radvc.conf`) by using a configuration like
+this:
+
+    interface bt0
+    {
+        AdvSendAdvert on;
+        prefix 2001:db8::/64
+        {
+            AdvOnLink off;
+            AdvAutonomous on;
+            AdvRouterAddr on;
+        };
```

Comment:
Please add

```
    abro 2001:db8:zzyy:xxff:feuu:vvww
    {
        AdvVersionLow 10;
        AdvVersionHigh 2;
        AdvValidLifeTime 2;
    };
```

Otherwise, the RIOT node will discard the RA, as it is in 6LN configuration (see https://tools.ietf.org/html/rfc6775#section-4.3).